### DISCUSSION TITLE
reduce the gap between the sections (tabs, breadcrumbs, and the editor)

### DISCUSSION THREAD

**Discussion by VikashRathod** [microsoft/vscode-discussions#1363] [Extension Development QnA]
![Screenshot from 2024-08-13 13-18-29](https://github.com/user-attachments/assets/3aee6d4a-c804-4005-9c54-66fc2d4bd0d2)
how can i get all 3 section in same line and reduce some space

**Comment by gjsjohnmurray** [comment]
This doesn't sound like an **extension development** question, so please see https://github.com/microsoft/vscode-discussions/discussions/5 for pointers about where to ask instead.

**Comment by theReynald** [comment]
Closing. Not related to extensions.
